invokeai/backend/model_management/seamless.py

```diff
@@ -28,68 +28,49 @@ def _conv_forward_asymmetric(self, input, weight, bias):
 def set_seamless(model: Union[UNet2DConditionModel, AutoencoderKL], seamless_axes: List[str]):
     try:
         to_restore = []
+        skipped_layers = 0
+        skip_second_resnet = True
+        skip_conv2 = True
```

Comment:
What's the point of these three vars? They're set to `0` and `True` always. They're not altered anywhere else in the function or pass into the function.

Reply:
Okay I see that it hardcodes vars from another branch. But I think you can just remove these from this PR and let the other branch actually take over when if and when it gets merged in.